sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -47,6 +48,21 @@
   DEFAULT_SAMPLING_PERIOD_MS = 0
 
 
+def retry(attempts):
+  def apply(f):
+    @functools.wraps(f)
+    def wrapper(*args):
+      for _ in range(attempts - 1):
```

Comment:
Do we need to return the value of decorated function or pass kwargs? I guess neither is relevant for test methods.